src/test/debuginfo/type-names.js

```diff
@@ -0,0 +1,10 @@
+"use strict";
+
+function getFunctionDetails(name)
```

Comment:
Can you add a comment explaining what this file is about?

Reply:
Sure, will do